github_pr.py

```diff
@@ -14,7 +14,7 @@ def _check_required_fields(required, **args):
 
 
 def _print_prs(prs, **args):
```

Comment:
really not a fan of how ```**args``` is used excessively in this module. Not really in the scope of this PR...